subsys/logging/Kconfig

```diff
@@ -68,6 +69,7 @@ config LOG_MAX_LEVEL
 	  - 4 DEBUG, maximal level set to LOG_LEVEL_DBG
 
 menu "Prepend log message with function name"
+	depends on !LOG_FRONTEND
```

Comment:
i think that it is possible to still prepend with function name. It may make less sense but i wouldn't disable that.